cc_proxy.go

```diff
@@ -205,10 +203,21 @@ func (p *ccProxy) sendCmd(cmd interface{}) (interface{}, error) {
 		return nil, fmt.Errorf("Wrong command type, should be hyperstartProxyCmd type")
 	}
 
-	err := p.client.Hyper(proxyCmd.cmd, proxyCmd.message)
+	data, err := json.Marshal(proxyCmd.message)
 	if err != nil {
 		return nil, err
 	}
 
+	var tokens []string
+	if proxyCmd.token == "" {
+		tokens = nil
+	} else {
+		tokens = append(tokens, proxyCmd.token)
+	}
+
+	if err := p.client.HyperWithTokens(proxyCmd.cmd, tokens, json.RawMessage(data)); err != nil {
```

Comment:
This is starting to get a bit off-topic for this PR, but every call to `sendCmd()` currently throws away the returned `interface{}`. Is that an oversight, or can we just drop this `interface{}` value until we come across a proxy that actually needs it?

Also, I'm not clear from the `proxy` interface how this value should actually be used. Maybe some more doc would be helpful here (not required as part of this PR though).